<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="dijkstraBox">
        <property name="text">
         <string>dijkstra</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bestBox">
        <property name="text">
         <string>best search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="depthBox">
        <property name="text">
         <string>depth search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="aBox">
        <property name="text">
         <string>a*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="scaleSpinBox">
        <property name="prefix">
         <string>scale: </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="upSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>up</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="downSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>down</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="leftSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>left</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rightSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>right</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="upLeftSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>up left</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="downLeftSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>down left</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="uprightSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>up right</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="downRightSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>down right</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="startRowSpinBox">
        <property name="prefix">
         <string>start row:  </string>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="startColumnSpinBox">
        <property name="prefix">
         <string>start column: </string>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="stopRowSpinBox">
        <property name="prefix">
         <string>stop row: </string>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="stopColumnSpinBox">
        <property name="prefix">
         <string>stop column: </string>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>startButton</zorder>
   <zorder>mdiArea</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
